docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
 +---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| CarSellOrderBox     | Box which contains car sell order data, which could be stored and operated in Sidechain.                                                                                                                              | boxData -- contains  car sell order data                                                                                                                                                                            |
+| CarSellOrderBox     | Box which contains car sell order data, which could be stored and operated in the Sidechain. ("...which is stored and managed..."?)                                                                                                                             | boxData -- contains  car sell order data                                                                                                                                                                            |
 +---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| CarSellOrderBoxData | Description of the car which is in sell status. That box data contains a special type of proposition SellOrderProposition. That proposition allows us to spent the box in two different ways: by seller and by buyer  | vin -- vehicle identification number which contains unique identification number of the car                                                                                                                         |
+| CarSellOrderBoxData | Description of the car which is in sell status. That box data contains a special type of proposition, a SellOrderProposition. That proposition allows us to spent(?) the box in two different ways: by seller and by buyer| vin -- vehicle identification number which contains the unique identification number of the car                                                                                                                         |
```

Comment:
@zennermac 

| CarSellOrderBoxData | Is the description of a vehicle for sale. The box data contains a special type of proposition, a SellOrderProposition. This proposition allows us to spend the box in two different ways: by seller and by buyer
| VIN -- vehicle identification number contains the unique identification number of the car                                                                                                                         |